tests/unit/Dns/Cname/Requests/AddRequestTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Tests\Unit\Dns\Cname\Requests;
+
+use PHPUnit\Framework\TestCase;
+use ResellerClub\Dns\Cname\Requests\AddRequest;
+use ResellerClub\TimeToLive;
+
+class AddRequestTest extends TestCase
+{
+    public function testInstantiation()
+    {
+        $ttl = TimeToLive::defaultTtl();
+        $request = new AddRequest(
+            'test.com',
```

Comment:
The request can be moved into the `setUp` and the values set to variables to make it obvious which ones are which.